oaknational/Oak-Web-Application
# feat: add new curriculum blog post

## Description

Music year: {owa_music_year}

- Updated the heading on the blog section
- Updated the curriculum blog posts to contain the one with the slug`our-new-curriculum-plans-are-here` as the uppermost
- Updated tests to include four blog posts

## Issue(s)

Fixes #[1037](https://www.notion.so/oaknationalacademy/Content-updates-to-Curriculum-landing-page-13726cc4e1b180699058f0885ef16a93)

## How to test

1. Go to {owa_deployment_url}
2. Visit `/teacher/curriculum`
3. You should see the new blog post on the top

## Screenshots

How it used to look (delete if n/a):
<img width="1211" alt="image" src="https://github.com/user-attachments/assets/816c41be-76ba-4c12-b942-898891bf8172">

How it should now look:
<img width="1341" alt="image" src="https://github.com/user-attachments/assets/1119294e-bdca-42e2-818b-6485ed6df86b">

## Checklist
